regulations/static/regulations/js/source/views/comment/attachment-view.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+var Backbone = require('backbone');
+Backbone.$ = $;
+
+var CommentEvents = require('../../events/comment-events');
+
+var AttachmentView = Backbone.View.extend({
+  events: {
+    'click .attachment-remove': 'handleRemove'
+  },
+
+  initialize: function(options) {
+    this.template = _.template(document.querySelector('#comment-attachment-template').innerHTML);
+    this.options = options;
+
+    if (this.options.xhr) {
+      this.options.xhr.upload.addEventListener('progress', this.handleProgress.bind(this));
+      this.options.xhr.upload.addEventListener('load', this.handleLoad.bind(this));
+    }
+
+    this.render();
+    this.$progress = this.$el.find('.attachment-progress');
+  },
+
+  render: function() {
+    var $el = $(this.template(this.options));
+    this.options.$parent.append($el);
+    Backbone.View.prototype.setElement.call(this, $el);
```

Comment:
Why use this rather than `this.setElement($el)`? Might be worth explaining in comments, at least.